code/modules/surgery/bodyparts/wounds.dm

```diff
@@ -67,6 +67,10 @@
 	if(HAS_TRAIT(owner, TRAIT_EASYDISMEMBER))
 		damage *= 1.1
 
+	//For inverse reagent Bamethan!
+	if(HAS_TRAIT(owner, TRAIT_BAMETHAN_BLEED) && ((woundtype == WOUND_PIERCE) || (woundtype == WOUND_SLASH)))
+		damage *= 1.5
```

Comment:
Consider making this a generic trait so other people can use it: `TRAIT_EASILY_WOUNDED_ONLY_BLEED_WOUNDS`, as a verbose example. 

Not only does it make it re-usable but also easier to understand what it is doing at a glance from the other side.